src/main/java/me/doubledutch/pikadb/PageFile.java

```diff
@@ -22,15 +23,15 @@ public PageFile(String filename) throws IOException{
 			ftest.delete();
 			wal=null;
 		}
+		this.cache=new SLRUPageCache(pageMap, 500);
```

Comment:
500 Pages sounds good for mobile usage... they are currently around 4kb each, so that would be around 2mb which seems a reasonable amount to me - however, it would be interesting while comparing the algorithms to run them with a big enough limit that they never have to evict pages and all we see in comparison are the overhead of the paging algorithm.

Once I/we have predicates fully implemented, we can do a more thorough test that would realistically exercise the algorithms on a constrained page set.